probe/docker/controls_test.go

```diff
@@ -59,7 +68,16 @@ func TestPipes(t *testing.T) {
 	mdc := newMockClient()
 	setupStubs(mdc, func() {
 		hr := controls.NewDefaultHandlerRegistry()
-		registry, _ := docker.NewRegistry(10*time.Second, nil, false, "", hr, "")
+		registry, _ := docker.NewRegistry(docker.RegistryOptions{
+			Interval:               10 * time.Second,
+			Pipes:                  nil,
```

Comment:
There is no need to initialize fields getting the default value (nil, false and  "")